### ISSUE TITLE
How to reboot VPS?

### ISSUE THREAD

**Issue by simonjackson1982** [kaosf/conoha#4]
Hi, 

We can stop and boot a VPS but how do we reboot a VPS? Do we have single cli? Thank you so much for your help.

Simon.

**Comment by kaosf** [comment]
I have implemented `conoha reboot` command just now, and released version 0.9.1.

Could you try to run the following command?

``` sh
gem install conoha
conoha -v #=> 0.9.1

conoha reboot yourUUID
```

You can select either "(normal) soft reboot" or "hard reboot" with 3rd argument specification like following:

``` sh
conoha reboot yourUUID # soft reboot
conoha reboot yourUUID hard # hard reboot
```

Thanks.

**Comment by simonjackson1982** [comment]
Thank you so much for your quick response!
